static/js/topic_page/map_tile.tsx

```diff
@@ -275,11 +279,18 @@ function draw(
   const getTooltipHtml = (place: NamedPlace) => {
     let value = "Data Unavailable";
     if (place.dcid in chartData.dataValues) {
-      value = formatNumber(
-        Math.round((chartData.dataValues[place.dcid] + Number.EPSILON) * 100) /
-          100,
-        props.statVarSpec.unit
-      );
+      // shows upto 4 decimal points for very low values
+      if (chartData.dataValues[place.dcid] < 0.01) {
+        const val = Number(chartData.dataValues[place.dcid]);
```

Comment:
nit: don't need the Number() since dataValues values are already numbers
nit: let's be more specific in the naming about what value this variable is referring to because we already have another variable named value.

Reply:
Done! Though I still had to do `Number(chartDatavalue.toPrecision(2))` in line 286, as `.Precision()` returned a string and we want a number!